components/tidb_query/src/rpn_expr/impl_string.rs

```diff
@@ -49,6 +49,15 @@ pub fn ascii(arg: &Option<Bytes>) -> Result<Option<i64>> {
     }))
 }
 
+#[rpn_fn]
+#[inline]
+pub fn reverse(arg: &Option<Bytes>) -> Result<Option<Bytes>> {
+    Ok(arg.as_ref().map(|bytes| {
+        let s = String::from_utf8_lossy(bytes);
```

Comment:
I think it is more reasonable to return an error if the data is corrupted (not valid UTF-8 bytes) rather than returning �.